src/OPLS.js

```diff
@@ -383,32 +388,33 @@ export class OPLS {
     }
   }
 
-  _predictAll(features, labels, options = {}) {
+  _predictAll(data, categories, options = {}) {
```

Comment:
I liked features as a manner to avoid data that is most confusing. If not necessary I would prefer to stick with features

Reply:
same for labels